1.8/nifi/README.md

```diff
@@ -0,0 +1,68 @@
+# How to use Apache Nifi on DC/OS
+
+[Nifi](http://nifi.apache.org) is an easy to use, powerful, and reliable system to process and distribute data.
+Running Apache Nifi on DC/OS allows you to manage your data flow very easile. Saving data on the HDFS or consuming data from Kafka will be easily scalable.
+
+- Estimated time for completion: 5 minutes
+- Target audience: Data engineers
+- Scope: Install and use Apache Nifi.
+
+**Table of Contents**:
+
+- [Prerequisites](#prerequisites)
+- [Install Nifi](#install-nifi)
+- [Use Nifi](#use-nifi)
+- [Uninstall Jenkins](#uninstall-jenkins)
```

Comment:
Fix the uninstall (should be nifi and not jenkins), please make sure that the links in the ToC actually work